### PR TITLE
1.9.2 release

### PR DESCRIPTION
**Pull request by godsflaw**
- [x] fix date format in index file
- [x] `kovan` address updates
- [x] `kovan` updated ABIs
- [x] `kovan` update `active` redirect
- [x] `kovan` spell passed
- [x] `kovan` PIP address updates
- [x] `kovan` IPFS file published
- [x] `mainnet` address updates
- [x] `mainnet` updated ABIs
- [x] `mainnet` update `active` redirect
- [x] `mainnet` spell passed
- [x] `mainnet` PIP address updates
- [x] `mainnet` IPFS file published

Diff of `kovan` address json:
```diff
150a151
>   "MIP21_LIQUIDATION_ORACLE": "0x2881c5dF65A8D81e38f7636122aFb456514804CC",
154d154
<   "MIP21_LIQUIDATION_ORACLE": "0x2881c5dF65A8D81e38f7636122aFb456514804CC",
163a164,187
>   "RWA003": "0xDBC559F5058E593981C48f4f09fA34323df42d51",
>   "PIP_RWA003": "0xA6f7FBeCef878a8B0Fa9AcB214040e962840f209",
>   "MCD_JOIN_RWA003_A": "0x4CCc7fED3912A32B6Cf7Db2FdA1554a9FF574099",
>   "RWA003_A_URN": "0x993c239179D6858769996bcAb5989ab2DF75913F",
>   "RWA003_A_INPUT_CONDUIT": "0x45e17E350279a2f28243983053B634897BA03b64",
>   "RWA003_A_OUTPUT_CONDUIT": "0x45e17E350279a2f28243983053B634897BA03b64",
>   "RWA004": "0x146b0abaB80a60Bfa3b4fDDb5056bBcFa4f1fec1",
>   "PIP_RWA004": "0xF8E535B9C1c230342EAdD1fe2636a872BdC3d8b4",
>   "MCD_JOIN_RWA004_A": "0xa92D4082BabF785Ba02f9C419509B7d08f2ef271",
>   "RWA004_A_URN": "0xf22C7F5A2AecE1E85263e3cec522BDCD3e392B59",
>   "RWA004_A_INPUT_CONDUIT": "0x303dFE04Be5731207c5213FbB54488B3aD9B9FE3",
>   "RWA004_A_OUTPUT_CONDUIT": "0x303dFE04Be5731207c5213FbB54488B3aD9B9FE3",
>   "RWA005": "0xcB2A48D26970eE7193d66BAc6F1b3090f2E8f82B",
>   "PIP_RWA005": "0x0dA25CE01BA1eCBD907694D5237Df0A2740Ce9E7",
>   "MCD_JOIN_RWA005_A": "0x1233d0DBb55A4Bb41D711d4B584f8DDB15A2Ff88",
>   "RWA005_A_URN": "0xdB9f0700EbBac596CCeF5b14D5e23664Db2A184f",
>   "RWA005_A_INPUT_CONDUIT": "0x17E5954Cdd3611Dd84e444F0ed555CC3a06cB319",
>   "RWA005_A_OUTPUT_CONDUIT": "0x17E5954Cdd3611Dd84e444F0ed555CC3a06cB319",
>   "RWA006": "0x4E65F06574F1630B4fF756C898Fe02f276D53E86",
>   "PIP_RWA006": "0x8A4B5c3fDe49486CDEda4D7Fbb4dec6CC6Af8258",
>   "MCD_JOIN_RWA006_A": "0x039B74bD0Adc35046B67E88509900D41b9D95430",
>   "RWA006_A_URN": "0x6fa6F9C11f5F129f6ECA4B391D9d32038A9666cD",
>   "RWA006_A_INPUT_CONDUIT": "0x652A3B3b91459504A8D1d785B0c923A34D638218",
>   "RWA006_A_OUTPUT_CONDUIT": "0x652A3B3b91459504A8D1d785B0c923A34D638218",
```

Diff of `mainnet` address json:
```diff
195a196
>   "MIP21_LIQUIDATION_ORACLE": "0x88f88Bb9E66241B73B84f3A6E197FbBa487b1E30",
199d199
<   "MIP21_LIQUIDATION_ORACLE": "0x88f88Bb9E66241B73B84f3A6E197FbBa487b1E30",
208a209,232
>   "RWA003": "0x07F0A80aD7AeB7BfB7f139EA71B3C8f7E17156B9",
>   "PIP_RWA003": "0xDeF7E88447F7D129420FC881B2a854ABB52B73B8",
>   "MCD_JOIN_RWA003_A": "0x1Fe789BBac5b141bdD795A3Bc5E12Af29dDB4b86",
>   "RWA003_A_URN": "0x7bF825718e7C388c3be16CFe9982539A7455540F",
>   "RWA003_A_INPUT_CONDUIT": "0x2A9798c6F165B6D60Cfb923Fe5BFD6f338695D9B",
>   "RWA003_A_OUTPUT_CONDUIT": "0x2A9798c6F165B6D60Cfb923Fe5BFD6f338695D9B",
>   "RWA004": "0x873F2101047A62F84456E3B2B13df2287925D3F9",
>   "PIP_RWA004": "0x5eEE1F3d14850332A75324514CcbD2DBC8Bbc566",
>   "MCD_JOIN_RWA004_A": "0xD50a8e9369140539D1c2D113c4dC1e659c6242eB",
>   "RWA004_A_URN": "0xeF1699548717aa4Cf47aD738316280b56814C821",
>   "RWA004_A_INPUT_CONDUIT": "0xe1ed3F588A98bF8a3744f4BF74Fd8540e81AdE3f",
>   "RWA004_A_OUTPUT_CONDUIT": "0xe1ed3F588A98bF8a3744f4BF74Fd8540e81AdE3f",
>   "RWA005": "0x6DB236515E90fC831D146f5829407746EDdc5296",
>   "PIP_RWA005": "0x8E6039C558738eb136833aB50271ae065c700d2B",
>   "MCD_JOIN_RWA005_A": "0xA4fD373b93aD8e054970A3d6cd4Fd4C31D08192e",
>   "RWA005_A_URN": "0xc40907545C57dB30F01a1c2acB242C7c7ACB2B90",
>   "RWA005_A_INPUT_CONDUIT": "0x5b702e1fEF3F556cbe219eE697D7f170A236cc66",
>   "RWA005_A_OUTPUT_CONDUIT": "0x5b702e1fEF3F556cbe219eE697D7f170A236cc66",
>   "RWA006": "0x4EE03cfBF6E784c462839f5954d60f7C2B60b113",
>   "PIP_RWA006": "0xB8AeCF04Fdf22Ef6C0c6b6536896e1F2870C41D3",
>   "MCD_JOIN_RWA006_A": "0x5E11E34b6745FeBa9449Ae53c185413d6EdC66BE",
>   "RWA006_A_URN": "0x0C185bf5388DdfDB288F4D875265d456D18FD9Cb",
>   "RWA006_A_INPUT_CONDUIT": "0x8Fe38D1E4293181273E2e323e4c16e0D1d4861e3",
>   "RWA006_A_OUTPUT_CONDUIT": "0x8Fe38D1E4293181273E2e323e4c16e0D1d4861e3",```

### CODE REVIEW

**Comment by kmbarry1** [comment]
Why was the position of the liquidation oracle changed?

**Comment by kmbarry1** [comment]
New addresses+keys for Kovan and `mainnet` look correct to me (modulo missing `mainnet` pips).

File: releases/mainnet/1.9.2/abi/index.html
Comment (kmbarry1) [inline releases/mainnet/1.9.2/abi/index.html:12]: This is a significant formatting change relative to 1.9.1:

https://github.com/makerdao/mcd-changelog/blob/master/releases/mainnet/1.9.1/index.html

Unless there's a good reason for this, let's minimize the diff. I guess deploying to IPFS (and hence having the IPFS link) is blocked on the pips, ofc.

Comment (kmbarry1) [reply releases/mainnet/1.9.2/abi/index.html:12]: Wait, nvm, I'm looking at the wrong file.

File: releases/mainnet/1.9.2/index.html
Comment (kmbarry1) [inline releases/mainnet/1.9.2/index.html:13]: Note to self: verify that IPFS link has changed once pips are available.

File: releases/kovan/1.9.2/index.html
Comment (kmbarry1) [inline releases/kovan/1.9.2/index.html:13]: This needs an update.

**Comment by godsflaw** [comment]
> Why was the position of the liquidation oracle changed?

I moved it to keep the `RWANNN` sections together.  Since the oracle is used for all of them, it seemed best to pull it out of the `RWA001` chunk.

This should be inconsequential programmatically, as the json file acts as a giant hash which would have non-deterministic positioning from language to language.

File: releases/kovan/1.9.2/index.html
Comment (godsflaw) [reply releases/kovan/1.9.2/index.html:13]: Wow, good catch.  I uploaded the new file yesterday, but it took > 30m to pin/mirror it and I forgot to cycle back and update this.

Comment (kmbarry1) [reply releases/kovan/1.9.2/index.html]: Confirmed that the updated IPFS link matches the Kovan contracts.json in this PR as of commit `d5abd4b`.

File: index.html
Comment (gbalabasquer) [inline index.html:54]: This date format thing was just discussed in the forums and LFW mentioned like the preferred one was YYYY-mm-dd (speaking about the forums).
I think it would be better to use that delimiter as it is the most used one for the international format.

File: vercel.json
Comment (gbalabasquer) [inline vercel.json:5]: In general we first merge a PR which keeps mainnet pointing to the prev version (which is finally changed when the spell is cast).
In this specific case I think we are fine leaving the PR unmerged until the spell actually passes as this is not something that integrators need to use.

File: releases/mainnet/1.9.2/index.html
Comment (zdumitru) [reply releases/mainnet/1.9.2/index.html:13]: IPFS link doesn't match mainnet contract addresses

File: releases/mainnet/1.9.2/index.html
Comment (zdumitru) [inline releases/mainnet/1.9.2/index.html:17]: GitHub release not available under that link. Either comment out or create release description

File: vercel.json
Comment (zdumitru) [reply vercel.json:5]: Should not be changed until mainnet release has been cast. This should be moved out to a separate PR.

File: vercel.json
Comment (zdumitru) [inline vercel.json:10]: Same comment as above

**Review by zdumitru — CHANGES_REQUESTED** [review]
Please see comments above and update accordingly.

File: releases/mainnet/1.9.2/index.html
Comment (godsflaw) [reply releases/mainnet/1.9.2/index.html]: updated now and mirroring it around.

File: vercel.json
Comment (godsflaw) [reply vercel.json:5]: updating the final bits we need (PIP addresses mainnet, IPFS mainnet, and changing the date delimiter) now that the mainnet spell is `cast()`

File: releases/mainnet/1.9.2/index.html
Comment (godsflaw) [reply releases/mainnet/1.9.2/index.html:17]: I'll wait for full signoff here so I can tag it.

File: vercel.json
Comment (godsflaw) [reply vercel.json:10]: yes, just waited for the PIP addresses from deploy this time.

**Comment by godsflaw** [comment]
Ready for final review once approved, and IPFS documents are mirrored, I will push the tag and merge.

File: releases/mainnet/1.9.2/index.html
Comment (kmbarry1) [reply releases/mainnet/1.9.2/index.html]: Contents of IPFS link match the addresses in this PR (including pips).

**Review by kmbarry1 — DISMISSED** [review]
LGTM

File: releases/mainnet/1.9.2/index.html
Comment (gbalabasquer) [reply releases/mainnet/1.9.2/index.html:17]: There won't be a 1.9.2 for the scripts, so this should be commented out.

Comment (godsflaw) [reply releases/mainnet/1.9.2/index.html:17]: Oh, yes, I missed that this was on deploy scripts.  I'll comment out.

Comment (godsflaw) [reply releases/mainnet/1.9.2/index.html:18]: You can look again, I pushed an update.

**Review by gbalabasquer — APPROVED** [review]
I haven't verified the new addresses are the correct ones but the addition of them and all the other changes look good to me.

**Review by kmbarry1 — APPROVED** [review]
LGTM

**Pull request merged**
